distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/opensearch/OpenSearchEndPoint.java

```diff
@@ -0,0 +1,73 @@
+package org.codice.ddf.itests.common.opensearch;
+
+import com.jayway.restassured.response.ValidatableResponse;
+
+import org.codice.ddf.itests.common.AbstractIntegrationTest;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+
+import static com.jayway.restassured.RestAssured.when;
+import static com.jayway.restassured.RestAssured.given;
+
+/**
+ * This class is used to perform open search query on a DDF end point
+ * that a client accesses to send query parameters and receive search results
+* */
+public class OpenSearchEndPoint {
```

Comment:
❗️ Instead of creating a new class, we may want to add this method to the existing `OpenSearchTestCommons` class instead.